### PR TITLE
fix: update documents for become contributors of Schrödinger Hat

### LINKED ISSUE #4: Change community levels
The base-level must be named as Member.

You can ask to become a Community Contributor.

### PR DESCRIPTION
**Pull request by Wabri**
Closes #4

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,10 +10,15 @@ The communication page lists communication channels like chat, issues, conferenc
...
+
+Start joining the Discord server and have a chat with the other folks.
 
 Each repo will often have a set of "help wanted" or "good first issue" issues that can help new contributors get involved.
 
 ## Membership
 
-We encourage all contributors to become members. We aim to grow an active, healthy community of contributors, reviewers and code owners. Learn more about requirements and responsibilities in our Community Membership page.
\ No newline at end of file
+We encourage to become ufficial contributors. We aim to grow an active, healthy community of contributors, reviewers and code owners. Learn more about requirements and responsibilities in our [Community Membership page](community-membership.md).
```
Comment (davideimola) [inline README.md]:
```suggestion
We encourage all members to become official contributors. We aim to grow an active, healthy community of contributors, reviewers, and code owners. Learn more about requirements and responsibilities on our [Community Membership page](community-membership.md).
```

File: community-membership.md [hunk 2]
```diff
@@ -4,17 +4,17 @@ This doc outlines the various responsibilities of contributor roles in Schrödin
...
+## Established contributors
 
-Established community members are expected to demonstrate their adherence to the principles in this document, familiarity with project organization, roles, policies, procedures, conventions, etc., and technical and/or writing ability. Role-specific expectations, responsibilities, and requirements are enumerated below.
+Established contributors are expected to demonstrate their adherence to the principles in this document, familiarity with project organization, roles, policies, procedures, conventions, etc. Role-specific expectations, responsibilities, and requirements are enumerated below.
 
-# Member
+# Contributors
 
-Members are continuously active contributors in the community. Members are expected to remain active contributors to the community.
+Contributors are continuously active in the community. Contributors are expected to remain active to the community.
```
Comment (davideimola) [inline community-membership.md]:
```suggestion
Contributors are continuously active members of the community. Contributors are expected to remain active in the community.
```

File: community-membership.md [hunk 3]
```diff
@@ -24,12 +24,12 @@ Members are continuously active contributors in the community. Members are expec
   - Filling or commenting on issues on GitHub
   - Contributing to project, or community discussion (e.g. meetings, Discord)
 - Have read the contributor guide
-- Sponsored by 2 members
+- Sponsored by 2 contributor
```
Comment (davideimola) [inline community-membership.md]:
```suggestion
- Sponsored by 2 contributors
```

File: community-membership.md [hunk 4]
```diff
@@ -24,12 +24,12 @@ Members are continuously active contributors in the community. Members are expec
...
 - Open an issue against this repository
   - Ensure your sponsors are @mentioned on the issue
   - Complete every item on the checklist
 - Have your sponsoring reviewers reply confirmation of sponsorship: `+1`
-- Once your sponsors have responded, your request will be reviewed by the Schrödinger Hat Admin team. Any missing information will be requested.
+- Once your sponsors have responded, your request will be reviewed by the Schrödinger Hat Contributors team. Any missing information will be requested.
```
Comment (davideimola) [inline community-membership.md]: I think we can't allow all the Contributors to make this.

We must have dedicated people for that with the following permissions:

- Add a Contributor inside the Discord Group
- Add a Contributor inside the GH org

File: community-membership.md [hunk 5]
```diff
@@ -38,7 +38,7 @@ Members are continuously active contributors in the community. Members are expec
   - Code is well tested
   - Tests consistently pass
   - Addresses bugs or issues discovered after code is accepted
-- Members can approve open PRs
+- Contributor can approve open PRs
```
Comment (davideimola) [inline community-membership.md]:
```suggestion
- Contributors can approve open PRs
```

File: community-membership.md [hunk 6]
```diff
@@ -76,12 +76,12 @@ The following apply to the projet for which one would be an owner.
 - Work with other project owners to maintain the project's overall health and success holistically
 
 
-# Inactive members
+# Inactive contributors
 
-_Members are continuously active contributors in the community._
+_Contributors are continuously active in the community._
```
Comment (davideimola) [inline community-membership.md]:
```suggestion
_Contributors are continuously active members of the community._
```

**Review by davideimola — CHANGES_REQUESTED** [review]

File: community-membership.md (hunk 4 above)
Comment (Wabri) [reply community-membership.md]: Reverted to Admin

**Review by davideimola — APPROVED** [review]
SGTM

**Pull request merged**